backend/src/test/java/ufrn/imd/boraPagar/user/UserTests.java

```diff
@@ -123,4 +133,22 @@ public void shouldPageOneHasLengthZero() {
 
         Assert.assertEquals(0, userPage.getContent().size());
     }
+
+    @Test
+    public void shouldPageZeroHasFirstTheObjectUserUsingOrderByName() {
+        Pageable pageable = PageRequest.of(0, 5);
+        Page<UserModel> userPage = repository.findAllByNameOrderByNameAsc(pageable);
+
+        Assert.assertEquals(user, userPage.getContent().get(0));
```

Comment:
Segue aquele mesmo outro ponto. Como que você garante que não tem outras disciplinas no banco de dados no momento de testar isso?

Reply:
Não entendi. Eu estou verificando se o primeiro elemento da minha página é o elemento `user`, que, afabéticamente vem primeiro.

Reply:
E se tiver um usuário do nada inserido no banco compartilhado por todos com o nome "a"? Como você vai previnir isso?

Reply:
Me pegou! Então eu devo verificar como?

Reply:
Pq o que eu quero testar é justamente que o primeiro elemento da página será o `Isaac`.

Reply:
Acredito que é melhor adicionar um banco local de testes mesmo.

Reply:
Gravei um vídeo tutorial no Youtube de como criar seu banco no Mongo Cloud.
https://youtu.be/LvEB7efljfQ

Reply:
Acredito que esse teste deva permanecer. Todos devem possuir um banco isolado mesmo.